File: Entidades/entidades/FormaPagamentoEnum.cs
using System.ComponentModel;

namespace Entidades.entidades
{
    public enum FormaPagamentoEnum
    {
        [Description("Dinheiro")]
        Dinheiro,

        [Description("Cartão de Crédito")]
        CartaoCredito,

        [Description("Cartão de Débito")]
        CartaoDebito,

        [Description("Pix")]
        Pix
    }
}

File: EasyPedidos_Flutter/lib/models/enums/forma_pagamento_enum.dart
enum FormaPagamentoEnum {
  dinheiro,
  cartaoCredito,
  cartaoDebito,
  pix,
  outros;

  String get description {
    switch (this) {
      case FormaPagamentoEnum.dinheiro:
        return 'Dinheiro';
      case FormaPagamentoEnum.cartaoCredito:
        return 'Cartão de Crédito';
      case FormaPagamentoEnum.cartaoDebito:
        return 'Cartão de Débito';
      case FormaPagamentoEnum.pix:
        return 'PIX';
      case FormaPagamentoEnum.outros:
        return 'Outros';
    }
  }
}
